yarn.lock

```diff
@@ -1761,6 +1761,15 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@babel/types@^7.4":
```

Comment:
are these new dependencies? Don't see anything in this PR that should require it

Reply:
These are dependencies for other packages. They could have bumped due to version change as well.

Reply:
okay 👍